hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -270,8 +277,28 @@ public StateView(
 	}
 
 	public Optional<ScheduleInfo> infoForSchedule(ScheduleID scheduleID) {
-		// TODO: Implement logic for getting information for schedule from store.
-		throw new UnsupportedOperationException();
+		try {
```

Comment:
We should update the `StateViewTest` with unit tests for the `infoForSchedule` and `scheduleExists` methods